docs/src/introduction.md

```diff
@@ -0,0 +1,28 @@
+# Introduction
+
+Feather is split into several smaller crates and projects:
+- `libcraft` contains data related to Minecraft and which is not directly related to feather.
+    - `blocks` contains all data related to Minecraft blocks. 
+    - `core` reexports all subcrates and contains data related to Minecraft which is not covered by any other subcrate.
+    - `generators` is used to generate `.rs` source files from `prismarinejs/minecraft-data`.
+    - `items` contains all data related to Minecraft items.
+    - `macros` contains all proc-macros used throughout `libcraft`.
+    - `particles` should maybe be moved to core, since it is a single file?
+- `feather`
+    - `base` is deprecated in favour of `libcraft`
+    - `blocks` is deprecated in favour of `libcraft`
+    - `common`
+    - `datapacks` should maybe be deprecated in favour of `libcraft` and be based on `prismarinejs/minecraft-data`.
```

Comment:
The datapacks crate can't/shouldn't be based on `minecraft-data`, but it will be moved to `libcraft`.